lib/gringotts/gateways/pin_payments.ex

```diff
@@ -162,6 +162,42 @@ defmodule Gringotts.Gateways.PinPayments do
     commit(:post, "charges", params)
   end
 
+  @doc """
+  Captures a previously authorised charge and returns its details. 
+
+  `amount` is transferred to the merchant account by PinPayments used in the
+  pre-authorization referenced by `payment_id`.
+
+  Captures a previously authorised charge and returns its details. 
+  Currently, you can only capture the full amount that was originally authorised. 
+
+  PinPayments returns a **Payment Id** which can be used later to:
+  * `refund/3` the amount.
+
+  ## Examples
+
+  The following example shows how one would capture a previously
+  authorized a payment worth $10 by referencing the obtained authorization `id`.
+  ```
+  iex> card = %CreditCard{first_name: "Harry",
+                          last_name: "Potter",
+                          number: "4200000000000000",
+                          year: 2099,
+                          month: 12,
+                          verification_code: "999",
+                          brand: "VISA"}
+  iex> money = Money.new(10000, :USD)
```

Comment:
The doc says this example captures $10 but your `Money` struct indicates a capture of Ten Thousand dollars.